### PR TITLE
bugfix/#5775

### PR DESCRIPTION
**Pull request by KKatiaa**
clear error, added translation

### CODE REVIEW

**Review by sobkiv — APPROVED** [review]

File: src/app/main/component/events/components/events-list/events-list.component.ts [hunk 1]
```diff
@@ -85,6 +85,7 @@ export class EventsListComponent implements OnInit, OnDestroy {
         this.elementsArePresent = this.eventsList.length < data.totalElements;
         this.eventLocationList = this.getUniqueCities(this.eventsList);
       }
+      console.log(res);
```
Comment (katia9261) [inline src/app/main/component/events/components/events-list/events-list.component.ts]: remove console.log

File: src/app/main/component/shared/components/events-list-item/events-list-item.component.ts [hunk 2]
```diff
@@ -116,6 +117,7 @@ export class EventsListItemComponent implements OnInit, OnDestroy {
     this.ecoEvents$.subscribe((res: IEcoEventsState) => {
       this.addAttenderError = res.error;
     });
+    console.log(this.event, '111');
```
Comment (katia9261) [inline src/app/main/component/shared/components/events-list-item/events-list-item.component.ts]: the same

File: src/app/main/component/shared/components/events-list-item/events-list-item.component.ts [hunk 3]
```diff
@@ -278,6 +281,17 @@ export class EventsListItemComponent implements OnInit, OnDestroy {
...
+    const sendEventDto = {
+      isFavorite: this.bookmarkSelected
+    };
+    const formData: FormData = new FormData();
+    const stringifiedDataToSend = JSON.stringify(sendEventDto);
+    const dtoName = 'EventPageResponceDto';
+
+    formData.append(dtoName, stringifiedDataToSend);
+    this.eventService.editEvent(formData);
+    console.log(this.event);
```
Comment (katia9261) [inline src/app/main/component/shared/components/events-list-item/events-list-item.component.ts]: the same

File: src/app/main/component/events/components/events-list/events-list.component.ts [hunk 4]
```diff
@@ -163,10 +164,6 @@ export class EventsListComponent implements OnInit, OnDestroy {
     this.searchToggle = !this.searchToggle;
   }
 
-  public showFavourite(): void {
```
Comment (katia9261) [inline src/app/main/component/events/components/events-list/events-list.component.ts:166]: as i see we use this function in HTML file, why did you delete it?

Comment (KKatiaa) [reply src/app/main/component/events/components/events-list/events-list.component.ts:166]: it was my function which i decided to delete because it is not finished

Comment (katia9261) [reply src/app/main/component/events/components/events-list/events-list.component.ts:166]: Maybe it makes sense to delete it also in the template?

**Review by ruslan-teh — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Review by Tadei-Mykola — APPROVED** [review]

**Review by OksanaFilipchuk — APPROVED** [review]

**Review by AlexW8719 — APPROVED** [review]
